speakersafetyd.service

```diff
@@ -4,6 +4,9 @@ Description=Speaker Protection Daemon
 [Service]
 Type=simple
 ExecStart=/usr/bin/speakersafetyd -c /usr/share/speakersafetyd/ -b /var/lib/speakersafetyd/blackbox -m 7
+User=speakersafetyd
```

Comment:
It might make sense to use [`DynamicUser`](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#DynamicUser=) for this, so distributions won't have to manage yet another static service user. Something like:

```
DynamicUser=yes
User=speakersafetyd
Group=speakersafetyd
RuntimeDirectory=/run/speakersafetyd
StateDirectory=/var/lib/speakersafetyd
```

(completely untested to be clear)

Reply:
Nay on dynamic user, speakersafetyd generates a "blackbox" file on crash, and services using dynamic user are not supposed to leave non-temporary files on exit.